### ISSUE TITLE
For me on Linux, the only way to resolve is by installing some dependencies:

### ISSUE THREAD

**Issue by ShnAlmighty** [puppeteer/puppeteer#8964]
For me on Linux, the only way to resolve is by installing some dependencies:

```bash
sudo apt-get update
```

```bash
sudo apt-get install -y gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```

```bash
sudo apt-get install -y libgbm-dev
```

[Source](https://www.ti-enxame.com/pt/puppeteer/headless-chrome-node-instalacao-da-api-e-do-apresentador-de-marionetes/806702850/)

_Originally posted by @tiagoboeing in https://github.com/puppeteer/puppeteer/issues/5662#issuecomment-732076246_

I guess some of these packages are not present in alpine based systems, is there something similar for alpine based systems?

_Originally posted by @ShnAlmighty in https://github.com/puppeteer/puppeteer/issues/5662#issuecomment-1247938727_

**Comment by OrKoN** [comment]
Please see https://pptr.dev/troubleshooting#running-on-alpine (happy to review a PR to update the troubleshoot doc if some instructions can be updated).

**Comment by ShnAlmighty** [comment]
> Please see https://pptr.dev/troubleshooting#running-on-alpine (happy to review a PR to update the troubleshoot doc if some instructions can be updated).

I integrated the changes in my Dockerfile but I am still getting `504 Gateway Time-out` error in my API response.

Also, the image I am using in Dockerfile is `node:14.16.1-alpine`.

I have tried changing the nginx config file also to allow more processing time for the API before sending response and also made the changes in the express app for the same but still getting the same error.
